CDP4ShellDialogs/Views/DataSourceSelection.xaml

```diff
@@ -116,10 +117,11 @@
                                                   Visibility="{Binding ElementName=ShowPasswordBox,
                                                        Path=IsChecked,
                                                        Converter={dx:BooleanToVisibilityConverter}}" />
-                                    <dxe:CheckEditBox x:Name="ShowPasswordBox"
+                                    <dxe1:SimpleButton x:Name="ShowPasswordBox"
```

Comment:
the new namespace needs to be removed as @alexatrhea suggested and this needs to be `<dx:SimpleButton`

Reply:
namespace removed and changed to dx